specification/storagepool/resource-manager/Microsoft.StoragePool/stable/2021-08-01/examples/IscsiTargets_Put.json

```diff
@@ -0,0 +1,102 @@
+{
+  "parameters": {
+    "api-version": "2021-08-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "diskPoolName": "myDiskPool",
+    "iscsiTargetName": "myIscsiTarget",
+    "iscsiTargetCreatePayload": {
+      "properties": {
+        "targetIqn": "iqn.2005-03.org.iscsi:server1",
+        "luns": [
+          {
+            "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
+            "name": "lun0"
+          }
+        ],
+        "aclMode": "Dynamic"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.StoragePool/diskPools/myDiskPool/iscsiTargets/myIscsiTarget",
+        "name": "myIscsiTarget",
+        "type": "Microsoft.StoragePool/diskPools/iscsiTargets",
+        "systemData": {
+          "createdAt": "2020-06-24T06:53:57+00:00",
+          "createdBy": "alias",
+          "createdByType": "User",
+          "lastModifiedAt": "2020-06-24T06:53:57+00:00",
+          "lastModifiedBy": "alias",
+          "lastModifiedByType": "User"
+        },
+        "properties": {
+          "targetIqn": "iqn.2005-03.org.iscsi:server1",
+          "luns": [
+            {
+              "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
+              "name": "lun0"
+            }
+          ],
+          "endpoints": [
+            "10.0.0.1:3260"
+          ],
+          "activeSessions": [
+            "iqn.2005-03.org.iscsi:client"
+          ],
+          "port": 3260,
+          "aclMode": "Dynamic",
+          "provisioningState": "Pending",
+          "status": "Unknown"
+        },
+        "managedBy": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.AVS/privateClouds/myPrivateCloud/clusters/Cluster-1/datastores/datastore1",
```

Comment:
>managedBy [](http://example.com/codeflow?start=9&length=9)

How's RBAC handled?

Reply:
We will require the caller RP to use S2S to obtain write access on this resource type.